src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -2,11 +2,9 @@ import React, { useState } from "react";
 import firebase from "firebase/app";
 import { useForm } from "react-hook-form";
 import { useHistory } from "react-router-dom";
-import axios from "axios";
```

Comment:
Since we aren't using `axios` here anymore, is there anywhere else we use it still? Can we remove it from our dependencies in `package.json` now? (see https://github.com/sparkletown/internal-sparkle-issues/issues/153)

Reply:
Yes we can it's not being used anywhere else.